rescript

```diff
@@ -135,6 +167,33 @@ The default \`rescript\` is equivalent to \`rescript build\` subcommand
 }
 
 var maybe_subcommand = process_argv[2];
+var is_building = false;
+function releaseBuild() {
+  if (is_building) {
+    try {
+      fs.unlinkSync(lockFileName);
+    } catch (err) {}
+    is_building = false;
```

Comment:
How do you know another build has not started now.

Reply:
`Another build` may come from the same process or an external process
For the same process, it is impossible since it it single threaded
For the external process, this does not matter